net/qBittorrent/Portfile

```diff
@@ -36,17 +36,19 @@ compiler.cxx_standard 2014
 compiler.blacklist-append *gcc-3.* *gcc-4.* {clang < 900.0.39.2}
 
 qt5.depends_component \
-                qttools
+                qttranslations
 
 configure.args-append \
                 --disable-gui
 
 destroot {
         if {[variant_isset gui]} {
-                exec ${qt_dir}/bin/macdeployqt ${worksrcpath}/src/qBittorrent.app -codesign=- >/dev/null 2>/dev/null
+                reinplace "s|Plugins = PlugIns|Plugins = ${qt_dir}/plugins|g" ${worksrcpath}/src/qBittorrent.app/Contents/Resources/qt.conf
+                exec /usr/bin/codesign --sign - ${worksrcpath}/src/qBittorrent.app
                 copy ${worksrcpath}/src/qBittorrent.app ${destroot}${applications_dir}
         } else {
-                exec ${qt_dir}/bin/macdeployqt ${worksrcpath}/src/qbittorrent-nox.app -codesign=- >/dev/null 2>/dev/null
+                reinplace "s|Plugins = PlugIns|Plugins = ${qt_dir}/plugins|g" ${worksrcpath}/src/qbittorrent-nox.app/Contents/Resources/qt.conf
+                exec /usr/bin/codesign --sign - ${worksrcpath}/src/qbittorrent-nox.app
```

Comment:
It looks like the only difference between the "if" and the "else" parts here are the app name. You could avoid the code duplication by just using the "if" and "else" to define a variable for the name of the app and then have the code just once referencing the variable.